cmd/skopeo/proxy.go

```diff
@@ -782,6 +784,8 @@ func (h *proxyHandler) processRequest(readBytes []byte) (rb replyBuf, terminate
 		err = fmt.Errorf("invalid request: %v", err)
 		return
 	}
+	logrus.Debugf("Executing method %s", req.Method)
```

Comment:
(For very broken or adversarial values, `%q` throughout would be better, but that’s not really relevant here.)

Reply:
Yeah...I guess I need to just internalize "use %q by default" in general.